### DISCUSSION TITLE
Welcome: introduce yourself!

### DISCUSSION THREAD

**Discussion by gianarb** [tinkerbell/tink#455] [General]
Hey! 

The Tinkerbell community is excited to have you here! ☕ 🍵 

Say hello and present yourself.

---

Let me start! I am Gianluca. I am a developer from Italy. I live nearby Turin at the foot of the Alps ⛰️ .
I work at Equinix Metal. I have a blog [gianarb.it](https://gianarb.it). When not working you can find me outside gardening and growing vegetables 🍅 🍈 🍓

**Comment by markjacksonfishing** [comment]
I’m Marky Jackson and I am a lover of family, friends, and a die-hard San Francisco Giants fan. I left my heart in San Francisco in the summer of 2020 and now reside in San Diego, California.

I am a senior software engineer at Equinix Metal, working on the Tinkerbell project.

I am a graduate student studying Data Science & Machine Learning under Michal Fabinger.

**Comment by gauravgahlot** [comment]
I'm Gaurav Gahlot, a Software Engineer from India 🇮🇳   working at [InfraCloud](https://www.infracloud.io/). I left Pune during Covid, and now live nearby Delhi with my family. 

I have seen Tinkerbell take off 🚀  from ideation to what it is now as I have been on the journey from Day 0. 😃 

I blog at [gauravgahlot.in](https://gauravgahlot.in). When I'm not working, I like to play guitar 🎸 or go for a ride 🚴🏼 .
